index.html

```diff
@@ -0,0 +1,485 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500;600;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles/style.css" />
+    <script src="main.js" defer></script>
+    <title>Adel Guitoun - Portfolio</title>
+  </head>
+  <body class="bg-decoration">
+    <!-- Header Section -->
+    <header class="header-section flex">
+      <nav class="nav-bar">
+        <button type="button" class="menu-btn">
+          <img src="./assets/icons/menu.svg" alt="menu" />
+          <img class="hidden" src="./assets/menu/cancel.svg" alt="cancel" style="color: red;"/>
+        </button>
+
+        <ul class="nav-list hidden" role="list">
```

Comment:
Kindly make sure that your mobile menu is displayed when the menu button is clicked🙏. Currently, clicking the menu button does not do anything.

**Hint:** You may need to double-check your javascript file and make sure you are selecting the correct elements from the DOM